docs/user_manual.md

```diff
@@ -272,7 +273,10 @@ For example, `{"foo": {"internalPort": 4711}}`  will  map the internal port
 4711 of the container to an arbitrary external port on the host. 
   
 Specifying `{"foo": {"internalPort": 4711, "externalPort": 80}}` will map
-internal port 4711  of the container to port 80 on the host. 
+internal port 4711  of the container to port 80 on the host.
+
+The `ip` attribute is optional and specifies the IP address on which the port
+will be exposed. Any IP string literal is valid.
```

Comment:
I think it would help to clarify that this is referring to an IP address of the host that the Helios job will run on, and there is no way for Helios to verify that this IP address actually exists or will work at deploy-time, and putting bad values here will cause your deploy to fail.